runner/process.py

```diff
@@ -255,7 +255,11 @@ def bins_errs(run, num_bins, max_error_locs):
         bbins = [[] for _ in range(num_bins)]
         bin_width = max_error_locs / num_bins
         for _ in range(BOOTSTRAP):
-            d = [len(random.choice(pexecs).costs) for pexecs in run.pexecs]
```

Comment:
Could also be
`d = [len(random.choice(pexecs).costs) for pexecs in run.pexecs if pexec.succeeded]`

Reply:
Ah no, mistook pexec for pexecs. Ignore this.